lounge/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+lounge:
+  image: thelounge/lounge:latest
+  environment:
+    PORT: 5000 # defaults to 9000
+  ports:
+    - "5000:5000"
```

Comment:
Yeah that might be a bit confusing. I thought it would show how one can change the port lounge listens to, which is pretty redundant unless you're running the container with `host` network mode. I'll update to `9000`.